servers/rendering/renderer_rd/storage_rd/mesh_storage.h

```diff
@@ -58,6 +59,12 @@ class MeshStorage : public RendererMeshStorage {
 		DEFAULT_RD_BUFFER_MAX,
 	};
 
+	enum AttributeLocation {
+		ATTRIBUTE_LOCATION_PREV_VERTEX = 12,
+		ATTRIBUTE_LOCATION_PREV_NORMAL = 13,
+		ATTRIBUTE_LOCATION_PREV_TANGENT = 14
+	};
```

Comment:
A nitpicky comment I know, but this should probably be private and not public as its not meant to be used outside of RD::MeshStorage

Reply:
Moved.